content/upload/upload_1.md

```diff
@@ -334,78 +323,435 @@ SELECT
+	EVENT_NAME,
+	EVENT_FORMAT,
+	EVENT_VERSION,
+	EVENT_FINGERPRINT,
+	CAST(TRUE_TSTAMP as TIMESTAMP) AS TRUE_TSTAMP,
+	CAST(LOAD_TSTAMP as TIMESTAMP) AS LOAD_TSTAMP,
+	FROM_JSON(REPLACE(REPLACE(CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_WEB_PAGE_1,'\"', ''),'''','\"'), 'array<struct<id:string>>') as CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_WEB_PAGE_1
+
+FROM ATOMIC.SAMPLE_EVENTS_ATTRIBUTION_BASE )
 
-Open the `snowflake_upload.py` file and edit the following variables.
 
-##### 2.1 Connection details - update username, password and account
-```python
-user = 'YOUR_USERNAME'
-password = 'YOUR_PASSWORD'
-account = 'YOUR_ACCOUNT'
 ```
 
-##### 2.2 Variables to be modified - warehouse and database
-```python
-warehouse='YOUR_WAREHOUSE'
-database = 'YOUR_DB
+***
+
+#### **Step 4:**  Drop the **SAMPLE_EVENTS_ATTRIBUTION_BASE** table
+
+```sql
+DROP TABLE ATOMIC.SAMPLE_EVENTS_ATTRIBUTION_BASE
 ```
-##### 2.3 Path to the sample_events_fractribution.csv
-```python
-csv_file = '/path/to/sample_events_fractribution.csv'
+You will now have the ATOMIC.SAMPLE_EVENTS_ATTRIBUTION created and loaded with sample data.
+
+
+{{% /tab %}}
+
+{{% tab name="BigQuery" %}}
+
+We have provided a public S3 bucket where we store the sample data in a csv format, you should be able to download it locally through this [link](https://snowplow-demo-datasets.s3.eu-central-1.amazonaws.com/Attribution_Modelling/Attribution_Modelling_sample_events.csv ).
+
+You can load the sample data to the warehouse using your [Google Cloud console](https://console.cloud.google.com/) as described in the below steps.
+
+For more details please check out the official [BigQuery documentation](https://cloud.google.com/bigquery/docs/batch-loading-data#console).
+
+
+#### **Step 1:**  Create the ATOMIC schema
+If the ATOMIC schema doesn't exist, create it in your target database.
+
+```sql
+CREATE SCHEMA IF NOT EXISTS ATOMIC
+
 ```
+***
+
+#### **Step 2:**  Upload the csv file
+
+2.1 Open the `BigQuery` page in the [Google Cloud console](https://console.cloud.google.com/).
+
+2.2 In the `Explorer` panel, expand your project and select the `atomic` schema/dataset. You should see the details panel open, if not, expand the `Actions` option (click on the three vertical dots) and click `Open`.
+
+2.3 In the `Details` panel, click `Create table`.
+
+3.4 On the `Create table` page fill out the sections:
+
+In the `Source` section:
+
+- For `Create table from`, select `Upload`.
+- For `Select file`, click `Browse`. Navigate to the `Attribution_Modelling_sample_events.csv` and open it.
+- Make sure `CSV` is selected for `File format`
+
+In the `Destination` section:
+
+On the `Create table` page,
+
+- For `Project`, choose the appropriate project (should be auto-populated).
+- For `Dataset` make sure it is `atomic` (should be auto-populated).
+- In the `Table` field put: `SAMPLE_EVENTS_ATTRIBUTION_BASE`
+Verify that Table type is set to `Native table`.
+
+On the `Schema` section:
+
+- Click `Auto-detect`
```

Comment:
We can load direct from S3 but this requires setting up a connection in BigQuery, so probably not worth it for this exercise.